JsonSchema/SchemaRegistry.cs

```diff
@@ -49,7 +49,10 @@ private class Registration
 			set => _fetch = value;
 		}
 
-		internal Draft ValidatingAs => _options.ValidatingAs;
+		internal Draft ValidatingAs
+		{
+			get => _options?.ValidatingAs ?? Draft.Unspecified;
```

Comment:
I'm more interested in giving why a registration can have a null options.  That shouldn't happen.

Reply:
I've explained this in #138 [here](https://github.com/gregsdennis/json-everything/issues/138#issuecomment-872855543). It's because of how the property is declared.

The following two property declarations are equivalent:

```csharp
internal Draft ValidatingAs => _options.ValidatingAs;

internal Draft ValidatingAs { get; } = _options.ValidatingAs;
```

Since this is a property with only a getter and an initialiser, the initialiser runs when the object is created, but before the constructor is called. The `_options` field does not get set until the constructor is called, meaning that it is still `null` when the property initialiser code runs.

Reply:
I think that, technically, the change I have made makes the null check redundant, because the property value is no longer being set before the constructor is called, meaning that `_options` will always be non-null by the time that the getter is called.

Reply:
Yeah I think I was too quick to blame the property initialiser syntax. Will let you dig into how/why `_options` can be `null`.